reporter/resources/buildings-points/transform.sql

```diff
@@ -0,0 +1,78 @@
+ALTER TABLE planet_osm_polygon ADD COLUMN "type" VARCHAR(255) NULL;
```

Comment:
This file violates DRY (dont repeat yourself). Can you find a way that it and others are included once only?